From e4658aad9622e40f474d3c3db9ff4e466fa79e45 Mon Sep 17 00:00:00 2001
From: Dan Murphy <dmurphy@ti.com>
Date: Fri, 4 Mar 2011 15:11:37 -0600
Subject: DISPC: Remove #ifdef from around the global

Remove the #ifdef from fake_vsync as when the flag is undefined
compilation breaks for.

The fake_vsync will always return false when the flag is undefined

Change-Id: Iab1de52730562d667bd319c4add110d2198916a2
Signed-off-by: Dan Murphy <dmurphy@ti.com>
---
 drivers/video/omap2/dss/dispc.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/drivers/video/omap2/dss/dispc.c b/drivers/video/omap2/dss/dispc.c
index ca1ee1c..98acfd5 100644
--- a/drivers/video/omap2/dss/dispc.c
+++ b/drivers/video/omap2/dss/dispc.c
@@ -54,9 +54,7 @@
 struct dispc_reg { u16 idx; };
 void __iomem  *dispc_base;
 
-#ifdef CONFIG_OMAP2_DSS_FAKE_VSYNC
 static int fake_vsync;
-#endif
 
 #define DISPC_REG(idx)			((const struct dispc_reg) { idx })
 
-- 
1.6.0.4

